service/src/integrationTest/resources/fixtures/caseDataWithEmptyStringsInsteadOfNull.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
this can also then be moved to the unit test resources

Reply:
resolving as we keeping in integration